CHANGELOG.md

```diff
@@ -11,6 +11,14 @@
 - Breaking changes
   - Remove `Context` parameter from `CriteoInterstitial` constructor
   - Remove `Parcelable` implementation from `AdSize` class
+  - Move `CriteoInterstitialAdDisplayListener#onAdFailedToDisplay(CriteoErrorCode)` to
+   `CriteoInterstitialAdListener#onAdFailedToReceive(CriteoErrorCode)`. This callback is fired
+    when any kind of error happen after requesting an interstitial ad.
+  - Move `CriteoInterstitialAdDisplayListener#onAdReadyToDisplay()` to
+   `CriteoInterstitialAdListener#onAdReceived(CriteoInterstitial)`. This callback is fired when an
+    interstitial ad is ready to be displayed.
+  - Provide default implementation in interstitial listener. Java 8 is required, see
+  https://developer.android.com/studio/write/java8-support.
```

Comment:
Let's make sure to sync with @irwansetiawan to update the documentation.

Reply:
Sure, I'm trying to condensate a quick summary here. But of course we might need to write a doc for helping on the migration.